src/widgets/Wizard.js

```diff
@@ -2,31 +2,46 @@
 /* eslint-disable react/forbid-prop-types */
 import React from 'react';
 import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
 
 import { Stepper } from './Stepper';
 import { TabNavigator } from './TabNavigator';
 import DataTablePageView from './DataTablePageView';
 
+import { WizardActions } from '../actions/WizardActions';
+import { selectCurrentTab } from '../selectors/wizard';
+
 /**
  * Layout component for a Tracker and TabNavigator, displaying steps
  * to completion for completion. See TabNavigator and StepsTracker
  * for individual component implementation.
  */
-export const Wizard = ({ tabs, titles, onPress, currentTabIndex }) => (
+const WizardComponent = ({ tabs, titles, currentTab, nextTab }) => (
   <DataTablePageView>
     <Stepper
       numberOfSteps={tabs.length}
-      currentStep={currentTabIndex}
-      onPress={onPress}
+      currentStep={currentTab}
+      onPress={nextTab}
       titles={titles}
     />
-    <TabNavigator tabs={tabs} currentTabIndex={currentTabIndex} />
+    <TabNavigator tabs={tabs} currentTabIndex={currentTab} />
   </DataTablePageView>
 );
 
-Wizard.propTypes = {
+WizardComponent.propTypes = {
   tabs: PropTypes.array.isRequired,
   titles: PropTypes.array.isRequired,
-  onPress: PropTypes.func.isRequired,
-  currentTabIndex: PropTypes.number.isRequired,
+  nextTab: PropTypes.func.isRequired,
+  currentTab: PropTypes.number.isRequired,
+};
+
+const mapStateToProps = state => {
+  const currentTab = selectCurrentTab(state);
+  return { currentTab };
 };
+
+const mapDispatchToProps = dispatch => ({
+  nextTab: tab => dispatch(WizardActions.switchTab(tab)),
```

Comment:
this is muddling the usage of `nextTab` which in some cases doesn't accept an argument now, and in this usage does. I'd suggest renaming this method / prop or changing it to use `WizardActions.NEXT_TAB`

Reply:
thanks ! 729fb1c904ca57b39110894b85e8439a11cab554